packages/next/next-server/server/normalize-page-path.ts

```diff
@@ -1,10 +1,14 @@
-import { posix } from 'path'
-
-export function normalizePathSep(path: string): string {
-  return path.replace(/\\/g, '/')
-}
+export { normalizePathSep, denormalizePagePath } from './denormalize-page-path'
 
 export function normalizePagePath(page: string): string {
+  let posix
+
+  // prevents path from being polyfilled in the browser since
+  // this method isn't used client side
+  if (typeof window === 'undefined') {
+    posix = require('path').posix
+  }
```

Comment:
This file is only used for this method, so I'm not sure I understand the necessity of this. Shouldn't this file only be imported by server code already (per "this method isn't used client side")?

Reply:
This is left over from trying to reduce the bundle size increase, ended up breaking the methods used client-side into a separate file, will revert since it's no longer needed